CHANGELOG.md

```diff
@@ -7,6 +7,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED]
 
+### Changed
+
+- Skipping stdout/stderr redirection tests until implemented in Dask parent process
+
+### Added
+
+- Simplifed starting the dask cluster using `multiprocessing`
+- Added `bokeh==2.4.3` to requirements.txt to enable view Dask dashboard
+
+## Fixed
```

Comment:
Could you fix the header here? This was from an earlier commit.

Reply:
fixed